### DISCUSSION TITLE
Merge continuous frames along the same track in stackStripMap.py

### DISCUSSION THREAD

**Discussion by leeyhgeo** [isce-framework/isce2#686] [Q&A] [answered]
I'm processing stacks of Envisat SAR images over a large region that requires three continuous Envisat frames along the same track to cover. For now, I perform stackStripMap.py for these three frames separately, and then use GIS software to mosaic the final velocity results that were computed via Mintpy.

However, I wonder if there is any solution to merge the original Envisat SAR images along the same track on the same date when running stackStripMap.py so that I can obtain a three-frame-long stack. I know that this function is available when doing a single-pair DInSAR via topsApp.py, but I have no idea if there is the same function and how to do so when using stackStripMap.py.

**Comment by fdelgadodelapuente** [comment, marked as answer]
Dump all the files ASA_IM_*.N1 files for each date in the same folder and then extract them with 
unpackFrame_ENV_raw.py  -i download/20070504 -o SLC/20070504 -m
The -m flag means merge all the raw frames. This does not work with SLC data.

**Reply by leeyhgeo** [reply]
Thank you, fdelgadodelapuente! I'll try this approach.
